test/reservation_test.rb

```diff
@@ -5,7 +5,9 @@
     it "returns a number" do
       start_date = Date.new(2017, 01, 01)
       end_date = start_date + 3
-      reservation = Hotel::Reservation.new(start_date, end_date, nil)
+      room_number = 7
+      rate = 200
+      reservation = Hotel::Reservation.new(start_date, end_date, room_number, rate)
       expect(reservation.cost).must_be_kind_of Numeric
```

Comment:
In addition to checking that cost is `Numeric` you should check that it is the correct value.